### DISCUSSION TITLE
这里能当成博客用么

### DISCUSSION THREAD

**Discussion by itorr** [itorr/itorr#1] [General]
ψ(｀∇´)ψ

<img width="805" alt="截屏2022-04-21 下午5 53 39" src="https://user-images.githubusercontent.com/1933673/178986158-c1fcc747-8705-4dc3-a5f3-def4563304e0.png">

https://user-images.githubusercontent.com/1933673/178986198-8df8bbb4-c856-474c-994e-a4aa1117fdf1.mov

![lab magiconch com 福音戰士標題生成器 -1657251489048](https://user-images.githubusercontent.com/1933673/178986355-f62bf5b1-aa82-4720-a02e-d3ad763f440a.jpg)

![IMG_7007](https://user-images.githubusercontent.com/1933673/178986407-950121a4-4897-4c62-94bf-e07fe1259dd3.JPG)

We don’t support that file type.  with a GIF, JPEG, JPG, MOV, MP4, PNG, SVG, WEBM, CSV, DOCX, FODG, FODP, FODS, FODT, GZ, LOG, MD, ODF, ODG, ODP, ODS, ODT, PDF, PPTX, TGZ, TXT, XLS, XLSX or ZIP.

![icon-3](https://user-images.githubusercontent.com/1933673/178986493-ff29d452-4c00-4242-ac12-e471cd598798.png)
![icon-1](https://user-images.githubusercontent.com/1933673/178986499-5421728c-f4ff-41b0-98bc-fa4dc550baa9.png)
![icon-2](https://user-images.githubusercontent.com/1933673/178986504-82cd0926-7bdf-4168-b1f9-6c99f6a1779e.png)
![icon-6](https://user-images.githubusercontent.com/1933673/178986507-189b5f56-59ac-4874-9f05-9a9b319dc4b3.png)
![icon-7](https://user-images.githubusercontent.com/1933673/178986511-d2f4d916-e332-47d5-bbb1-92a25dc840f5.png)
![icon-4](https://user-images.githubusercontent.com/1933673/178986512-4b0f45b7-9263-4ab4-908e-4366b1fc6356.png)
![icon-5](https://user-images.githubusercontent.com/1933673/178986515-3580ea99-aa2e-4e8d-89c5-ebb5767c89d7.png)

**Comment by itorr** [comment]
我感觉不太行😅

**Reply by itorr** [reply]
- [ ] 还有二级评论
待办事项可以快速编辑

**Comment by Remtg** [comment]
問題ないよ
